### ISSUE TITLE
Component gallery SR descriptions 

### ISSUE THREAD

**Issue by GCHQ-Developer-465** [mi6/ic-design-system#718]
## Summary of the bug
This is a problem with the Voiceover screen reader when navigating to card within the component gallery. There should be a pause such as a comma in between the component name and description. This is highlighted in [component gallery](https://design.sis.gov.uk/components). 

The gallery cards are announced as "Link, badge Badges are attached...'. This rapid repetition of the link title and the description could be confusing for a user of a screen reader. 

An additional problem occurs with NVDA as when each card is focussed, only the link is announced, and not the card description. For example, when navigating to the badge card the following is announced:

"Visited link, clickable, badge"

## 🪜 How to reproduce

Tell us the steps to reproduce the problem:

Problem 1
1. Go to page the [component gallery page](https://design.sis.gov.uk/components) for instance as this is more visible there.
2. Use the Voiceover screen reader
3. See the duplication of the component name and the lack of pause.  (EDIT: pause issue now covered by #607)

Problem 2 - EDIT: now covered by: https://github.com/mi6/ic-ui-kit/issues/1486
1. Go to page the [component gallery page](https://design.sis.gov.uk/components).
2. Use the NVDA screen reader
3. Navigate to one of the gallery cards using either the virtual cursor or the tab key. 
4. See that only the link is announced and not the card description. 

## 📸 Screenshots or code

<img width="514" alt="Screenshot 2023-12-07 at 09 34 12" src="https://github.com/mi6/ic-design-system/assets/111882114/5fba2449-e416-441b-83f1-7adcc2b6e3ad">

## 🧐 Expected behaviour

Each card should append the term "component" to the visible link's accessible label, so that instead of it being announced only using the component name such as "Link, badge" it is read as "Link, Badge component". 

To adhere to WCAG criteria 'Label In Name' the visible label must be contained within the accessible label. This would be achieved using the below format:

Accessible label = [Visible label] + " component."

If possible, a pause should also be used between the link announcement and the card description. Punctuation added to the accessible label may support this. 

*Given* that a card in the component gallery is navigated to with a Voiceover SR
*When* it is announced, 
*Then* it should be announced as follows:
"Link, [visible label] + " component.", [card description]."

*Given* that a card in the component gallery is navigated to with NVDA SR, 
*When* it is navigated to using either virtual cursor or tab focus, 
*Then* its contents should be announced in full, including the description.

**Comment by ad9242** [comment]
Scope of this ticket is now just to append "component" into the accessible label

the issue re adding a pause will be covered by #607 

the issue re the description to be read out in NVDA, is now covered by
